### PR TITLE
PR for sync states

### PR DESCRIPTION
**Pull request by sowmyaraj92**
Added a flow diagram
Modified the syncing states for version control

### CODE REVIEW

File: src/pages/docs/getting-started/syncing.md [hunk 1]
```diff
@@ -67,15 +67,29 @@ Once you log in to Postman app, your data gets synced across all devices. Postma
 
 ### States
 
-If your app is in sync with the server, the icon to the left of your name in the top navigation appears orange, and displays **IN SYNC**.
+Syncing states helps you to understand the sync status of your workspace and enables you to take actions whenever necessary. The flow diagram for sync states is shown below:
```
Comment (SueSmith) [inline src/pages/docs/getting-started/syncing.md:70]: I'd reword to something simpler like "Postman indicates the sync state of your workspace." and just leave the second sentence out..

File: src/pages/docs/getting-started/syncing.md [hunk 2]
```diff
@@ -67,15 +67,29 @@ Once you log in to Postman app, your data gets synced across all devices. Postma
...
 
-If a sync operation is underway, it displays **SYNCING**.
+<img src="https://assets.postman.com/postman-docs/sync-states-flow.jpg" alt="Sync States" width="850px"/>
```
Comment (SueSmith) [inline src/pages/docs/getting-started/syncing.md:72]: for most imgs we just use markdown i.e. `Alt text` - would only use html img tags if you need the image to display at less than ~600px width

File: src/pages/docs/getting-started/syncing.md [hunk 3]
```diff
@@ -67,15 +67,29 @@ Once you log in to Postman app, your data gets synced across all devices. Postma
...
+If you are not signed in to the app, you see the state as **OFFLINE**.
+
+<img src="https://assets.postman.com/postman-docs/offline-state.jpg" alt="Offline state" width="850px"/>
 
 When you open the app or if your connection drops, it displays **CONNECTING** before it starts to sync.
 
-[![sync icon in header](https://assets.postman.com/postman-docs/Sync-Screen.png)](https://assets.postman.com/postman-docs/Sync-Screen.png)
+<img src="https://assets.postman.com/postman-docs/connecting-state.jpg" alt="Connecting state" width="850px"/>
+
+If you hover on the icon, you see **SYNCING**. The popup will show the information about the current state.
```
Comment (SueSmith) [inline src/pages/docs/getting-started/syncing.md:82]: * "hover over"
* "you will see **Syncing**"
* "The popup will indicate further information about the current sync state."

File: src/pages/docs/getting-started/syncing.md [hunk 4]
```diff
@@ -67,15 +67,29 @@ Once you log in to Postman app, your data gets synced across all devices. Postma
...
+
+<img src="https://assets.postman.com/postman-docs/online-state-syncing.jpg" alt="Online state syncing" width="850px"/>
+
+If your app is in sync with the server, the icon to the left of your name in the top navigation appears online, and displays a popup showing **All changes are saved** on hover.
```
Comment (SueSmith) [inline src/pages/docs/getting-started/syncing.md:86]: * "in the top navigation" > "at the top right"
* "appears online" > "indicates **Online**"

File: src/pages/docs/getting-started/syncing.md [hunk 5]
```diff
@@ -67,15 +67,29 @@ Once you log in to Postman app, your data gets synced across all devices. Postma
...
+
+<img src="https://assets.postman.com/postman-docs/online-state-in-sync.jpg" alt="Online state in sync" width="850px"/>
+
+While connecting to the app, if you encounter a syncing error, it goes to **ERROR** state. The popup will now display the detailed information on the error.
```
Comment (SueSmith) [inline src/pages/docs/getting-started/syncing.md:90]: * "it goes to" > "Postman will indicate"
* for second sentence I'd do something like "Hover to see detailed information on the error."

File: src/pages/docs/getting-started/syncing.md [hunk 6]
```diff
@@ -67,15 +67,29 @@ Once you log in to Postman app, your data gets synced across all devices. Postma
...
 ### States
 
-If your app is in sync with the server, the icon to the left of your name in the top navigation appears orange, and displays **IN SYNC**.
+Syncing states helps you to understand the sync status of your workspace and enables you to take actions whenever necessary. The flow diagram for sync states is shown below:
 
-If a sync operation is underway, it displays **SYNCING**.
+<img src="https://assets.postman.com/postman-docs/sync-states-flow.jpg" alt="Sync States" width="850px"/>
 
-If you are not signed in to the app, you see **OFFLINE**.
+If you are not signed in to the app, you see the state as **OFFLINE**.
```
Comment (SueSmith) [inline src/pages/docs/getting-started/syncing.md:74]: it looks like these states aren't displayed in all caps in the ui anymore so can we switch them to initial caps only e.g. "Offline" etc throughout?

File: src/pages/docs/getting-started/syncing.md (hunk 1 above)
Comment (sowmyaraj92) [reply src/pages/docs/getting-started/syncing.md]: Done

File: src/pages/docs/getting-started/syncing.md (hunk 2 above)
Comment (sowmyaraj92) [reply src/pages/docs/getting-started/syncing.md]: done

File: src/pages/docs/getting-started/syncing.md (hunk 3 above)
Comment (sowmyaraj92) [reply src/pages/docs/getting-started/syncing.md]: done

File: src/pages/docs/getting-started/syncing.md (hunk 4 above)
Comment (sowmyaraj92) [reply src/pages/docs/getting-started/syncing.md]: done

File: src/pages/docs/getting-started/syncing.md (hunk 5 above)
Comment (sowmyaraj92) [reply src/pages/docs/getting-started/syncing.md]: done

File: src/pages/docs/getting-started/syncing.md (hunk 6 above)
Comment (sowmyaraj92) [reply src/pages/docs/getting-started/syncing.md]: done

**Comment by sowmyaraj92** [comment]
> few wee requests @sowmyaraj92 give me a shout if you need any more detail!

>The changes have been made :)

File: src/pages/docs/getting-started/syncing.md [hunk 7]
```diff
@@ -67,15 +67,29 @@ Once you log in to Postman app, your data gets synced across all devices. Postma
...
 
-When you open the app or if your connection drops, it displays **CONNECTING** before it starts to sync.
+[![Offline state](https://assets.postman.com/postman-docs/offline-state.jpg)](https://assets.postman.com/postman-docs/offline-state.jpg)
 
-[![sync icon in header](https://assets.postman.com/postman-docs/Sync-Screen.png)](https://assets.postman.com/postman-docs/Sync-Screen.png)
+When you open the app or if your connection drops, it displays **Connecting** before it starts to sync.
+
+[![Connecting state](https://assets.postman.com/postman-docs/connecting-state.jpg)](https://assets.postman.com/postman-docs/connecting-state.jpg)
+
+Hover over the icon, you will see **Syncing**. The popup will indicate further information about the current sync state.
```
Comment (SueSmith) [inline src/pages/docs/getting-started/syncing.md:82]: For the first sentence I just meant "If you hover over..." or "When you hover over..." would work too

**Comment by DannyDainton** [comment]
Hey @sowmyaraj92 

Are we sure that this is the design we're going within the app? 

Artemis has a slightly different styling/behaviour for the different states.

Screenshot 2020-07-22 at 08 23 04

**Comment by sowmyaraj92** [comment]
> Hey @sowmyaraj92
> 
> Are we sure that this is the design we're going within the app?
> 
> Artemis has a slightly different styling/behaviour for the different states.
> 
> Screenshot 2020-07-22 at 08 23 04

**Review by arlemi — APPROVED** [review]
Looks good to me! 🚀
